plugin.rb

```diff
@@ -14,28 +14,15 @@ module ::DiscourseCustomTopicLists
 end
 
 require_relative "lib/discourse_custom_topic_lists/engine"
+require_relative "lib/discourse_custom_topic_lists/custom_topic_list"
 register_asset "stylesheets/common/common.scss"
 
 after_initialize do
+  require_relative "app/controllers/discourse_custom_topic_lists/lists_controller"
```

Comment:
Can you see if you can update the plugin to take advantage of Rails' autoloading? https://meta.discourse.org/t/structuring-a-plugin-for-rails-autoloading/256092 

Manually requiring files should not be required.